docs/develop/requirements-property-unified-manifest.md

```diff
@@ -0,0 +1,203 @@
+            ...
+            {
+                ...
+                "events": {
+                    "type": "OnMailSend",
+                    "actionId": "logOutgoingEmail",
+                    "options": {
+                        "sendMode": "promptUser"
+                    }
+                },
+                "requirements": {
+                    "capabilities": [
+                        {
+                            "name": "MailBox",
+                            "minVersion": "1.13"
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+]
+```
+
+### extensions.ribbons.requirements
+
+The "extensions.ribbons" property is used to customize the Office application ribbon when the add-in is installed. The "requirements" subproperty can be used to prevent the customizations in some versions of Office.
+
+For example, suppose an Outlook add-in is configured to add a custom button to the ribbon and the button runs a function that uses code introduced in the **Mailbox 1.9** requirement set. But the add-in has other features that would be useful on versions of Office that only support version 1.8. To ensure that the add-in is installable on versions that support 1.8, a developer can set the “extensions.requirements.capabilities” property to the requirement set **MailBox 1.8** instead of 1.9. But to block the custom button from appearing on the ribbon in versions that don't support 1.9, the developer can add an “extensions.ribbons.requirements.capabilities” property that specifies **Mailbox 1.9**. The following is an example. For details of the custom ribbon configuration, see [Create add-in commands with the unified manifest for Microsoft 365](create-addin-commands-unified-manifest.md).
+
+```json
+"extensions": [
+    ...
+    {
+        ...
+        "ribbons": [
+            ...
+            {
+                // Details of the ribbon configuration would be here.
+
+                "requirements": {
+                    "capabilities": [
+                        {
+                            "name": "MailBox",
+                            "minVersion": "1.9"
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+]
+```
+
+### extensions.runtimes.requirements
+
+The "extensions.runtimes" property configures the sets of runtimes and actions that each extension point can use. For more information on its usage, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command). For more information about runtimes in Office add-ins, see [Runtimes in Office Add-ins](../testing/runtimes.md).
```

Comment:
If this article is going to live in another repo, then you should update all the "internal" links from being relative in this repo e.g.
```suggestion
The "extensions.runtimes" property configures the sets of runtimes and actions that each extension point can use. For more information on its usage, see [create add-in commands](/office/dev/add-ins/develop/create-addin-commands-unified-manifest), [configure the runtime for a task pane](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-task-pane-command), and [configure the runtime for the function command](/office/dev/add-ins/develop/create-addin-commands-unified-manifest#configure-the-runtime-for-the-function-command). For more information about runtimes in Office add-ins, see [Runtimes in Office Add-ins](/office/dev/add-ins/testing/runtimes).
```